proteus/mprans/SpatialTools.py

```diff
@@ -83,7 +84,7 @@ def _attachAuxiliaryVariable(self, key):
         """
         if key not in self.auxiliaryVariables:
             if key == 'RigidBody':
-                self.auxiliaryVariables[key] = True
+                self.auxiliaryVariables[key] = bd.RigidBody(self)
```

Comment:
Lots of codes like to mess around with `dt_init` and `cfl_target` values as well.  Should Shapes hold onto the default values (instead of, or in addition to, how BodyDynamics is the only one with control over them currently), pick up user defined values when initialized in the user code (as an optional thing to specify for a shape) or when the rigid body is declared, and pass them through here?  Or do you plan on having the user access those values directly from BodyDynamics via some method there?